src/docs/product/crons/feedback.mdx

```diff
@@ -0,0 +1,6 @@
+---
+title: Feedback
+sidebar_order: 4
+---
+
+Have trouble or want to provide feedback? Create an issue in our [Sentry Github repository](https://github.com/getsentry/sentry/issues/new/choose), or reach out to us at [crons-feedback@sentry.io](mailto:crons-feedback@sentry.io).
```

Comment:
```suggestion
If you're having trouble, or want to share feedback, we want to hear from you. Create an issue in our [Sentry Github repository](https://github.com/getsentry/sentry/issues/new/choose), or reach out to us at [crons-feedback@sentry.io](mailto:crons-feedback@sentry.io).
```

I would add this section to the bottom of the "troubleshooting page". It doesn't really make sense for this to have its own page.